PowerShell Scanners/Get Available Windows Updates/Get Available Windows Updates.ps1

```diff
@@ -1,6 +1,90 @@
 & '.\Install and Import Module.ps1' -ModuleName "PSWindowsUpdate"
 
-# The PowerShell Scanner currently does not understand the Collection object this cmdlet emits,
-# so we have to assign it to a variable to get PowerShell to unwrap it for us.
+function ConvertTo-Bytes {
```

Comment:
You could simply this down to just `Invoke-Expression $_.Size` so long as you don't fear the Size property could be abused. Because this is essentially opening yourself up to the PowerShell equivalent of a SQL injection.

Reply:
Wow, that's way simpler! Thanks for catching that :)

I agree that Invoke-Expression can be dangerous, but the data that it's working with here is provided by the Windows Update API. If an attacker has access to that, they could do far worse.